doozer/doozerlib/cli/images_streams.py

```diff
@@ -831,6 +832,20 @@ def search_issues(query):
             new_issues[distgit_key] = issue
             print(f'A JIRA issue has been opened for {pr.html_url}: {issue.key}')
             connect_issue_with_pr(pr, issue.key)
+            try:
+                # update created issue by missing custom fields
+                # Retrieve the issue object
+                issue = jira_client.issue(issue_key)
+                fields_to_add = {
+                    'fields': {
+                        'customfield_12317313': 'N/A',  # customfield_12317313 is Release Notes Text in JIRA
+                        'customfield_12320850': {'value': 'Release Note Not Required'},  # customfield_12320850 is Release Notes Type in JIRA
+                    }
+                }
+                # Use the update_issue method to update the issue
+                issue.update(fields=fields_to_add['fields'])
```

Comment:
How does jira behave if you `issue.update` with something it already has? Is it a no-op, or get followers of that issue an email notification? 
What happens if someone clicks this to a different value for whatever reason. The bot will override what a human says, correct? 
IMHO, the bot should only update this if it is not already set.

Reply:
it overwrites it - this was tested.